xds/internal/balancer/cdsbalancer/cdsbalancer.go

```diff
@@ -193,6 +221,7 @@ func (b *cdsBalancer) run() {
 					}
```

Comment:
Would it make sense to move everything in `case *ccUpdate` into a function (method on `cdsBalancer`)?